src/lazy/expanded/struct.rs

```diff
@@ -283,6 +282,18 @@ impl<'top, D: Decoder> LazyExpandedStruct<'top, D> {
                     }
                 }
             }
+            // For any other kind of struct, do a linear scan over its fields until we encounter
+            // one with the requested name.
```

Comment:
```suggestion
            // For any other kind of struct (e.g. struct literal or make_struct call),
            // do a linear scan over its fields until we encounter one with the requested name.
```

I like a reminder of what the other cases are. It doesn't have to come this way.

Reply:
I'm going to incorporate this in another PR I'm about to open.